starter-code/src/viking.js

```diff
@@ -1,8 +1,29 @@
-// Soldier
-function Soldier() {}
+//Soldier
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+};
+
+Soldier.prototype.attack = function(){
+return this.strength;
+}
+
+
+Soldier.prototype.receiveDamage = function(theDamage){
+ this.health=this.health-theDamage
+}
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+this.name = name
```

Comment:
Para heredar propiedades del constructor padre, deben utilizarlo de esta forma

function Viking(name, health, strength) {
    this.name = name;
    Soldier.call(this, health, strength);
}